server/consumer.go

```diff
@@ -399,6 +399,12 @@ func setConsumerConfigDefaults(config *ConsumerConfig, lim *JSLimitOpts, accLim
 	if config.DeliverSubject == _EMPTY_ && config.MaxRequestBatch == 0 && lim.MaxRequestBatch > 0 {
 		config.MaxRequestBatch = lim.MaxRequestBatch
 	}
+	if config.MaxAckPending == 0 {
+		config.MaxAckPending = streamCfg.LimitMaxAckPending
```

Comment:
Does streamCfg always have these set? Should they fall back to the server limits, etc?